randovania/games/dread/json_data/Artaria.txt

```diff
@@ -5344,15 +5390,18 @@ Extra - asset_id: collision_camera_082
           Any of the following:
               Gravity Suit
               Heat/Cold Runs (Intermediate) and Cold Damage ≥ 50
-          Walljump (Beginner) or Simple IBJ or Use Spin Boost
+          Any of the following:
+              Walljump (Beginner) or Simple IBJ or Use Spin Boost
+              Flash Shift and Speed Booster
```

Comment:
Where would you charge the speedbooster from this door? Does it require frozen artaria?

Reply:
You charge it at the bottom of the room/Red Chozo Arena. Jump up the first ledge, jump up against the wall, and spark diagonally then flash to ledge. But seeing as how beginner walljumps (with no other items for assistance) are being treated as trivial now, that will no longer matter. I will make this and all other walljumps i've done into trivial after the pr is in